src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx

```diff
@@ -214,6 +214,7 @@ export default class PublishCollectionModal extends React.PureComponent<Props, S
             {t('global.publish')}
           </Button>
           <p>{t('publish_collection_modal.accept_by_publishing')}</p>
+          {unsyncedCollectionError && <p className='error'>{t('publish_collection_modal.unsynced_collection')}</p>}
```

Comment:
Should we also prevent the publish button from working when this error is found?